<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>341</width>
      <height>391</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>2</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <row/>
    <row/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>AddRow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DellRow">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DellRow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AddColumn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>AddColumn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DelColumn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DelColumn</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxRow">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>50</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_GetSum">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getsum</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSum">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>sum</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetRand">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SetRand</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
